.trybe/requirements.json

```
@@ -0,0 +1,88 @@
+{
+  "requirements": [
+    { 
+      "description": "1 - Crie uma barra verde na parte superior da página",
+      "bonus": false
+    },
+    { 
+      "description": "2 - A barra superior deve conter o logotipo do Trybewarts no canto esquerdo com a classe trybewarts-header-logo",
+      "bonus": false
+    },
+    { 
+      "description": "3 - A barra superior deve conter um formulário de autenticação no canto direito com um input de login, um de senha e um botão de entrar",
```

```suggestion
      "description": "3 - Acrescente um formulário de autenticação no canto direito da barra superior contendo os inputs de login, de senha e um botão de entrar",
```
Mesmo situação do anterior. Coloquei neste formato, mas ainda não fiquei contente com ele rsrs, vejam o que acham.